Voycar.Api.Web/Program.cs

```diff
@@ -124,6 +125,9 @@
                });
                ep.Summary(s =>
                {
+                   var existingDescription = s.Description;
+                   var rolesDescription = $"\n\nRequired role: {(ep.AllowedRoles.Count > 0 ? string.Join(", ", ep.AllowedRoles) : "")}";
+                   s.Description = existingDescription + rolesDescription;
```

Comment:
Kann man das nicht auch so machen?
```suggestion
                   var s.Description += $"\n\nRequired role: {(ep.AllowedRoles.Count > 0 ? string.Join(", ", ep.AllowedRoles) : "")}";
```

Reply:
Hm ja jetzt geht es so. Halt ohne das "var". Hab ich angepasst.

Reply:
![grafik](https://github.com/solid-stack-solutions/voycar-backend/assets/136514166/37aeda86-2d27-4220-bee1-990a3b75f94c)